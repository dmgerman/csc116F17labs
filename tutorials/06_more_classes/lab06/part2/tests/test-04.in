3 3
1 2 3
4 5 6
7 8 9

2 2
1 1
2 2