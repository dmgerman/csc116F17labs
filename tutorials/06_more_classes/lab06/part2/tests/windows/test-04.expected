Reading first matrix
Enter the number of rows and columns: 
Enter the elements of the matrix: 

Successfully read a matrix with dimensions 3 x 3
M1: 
1 2 3 
4 5 6 
7 8 9 
Reading second matrix
Enter the number of rows and columns: 
Enter the elements of the matrix: 

Successfully read a matrix with dimensions 2 x 2
M2: 
1 1 
2 2 
M1 == M2? false
M1 != M2? true
2*M1:
2 4 6 
8 10 12 
14 16 18 
3*M1:
3 6 9 
12 15 18 
21 24 27 
M1*4:
4 8 12 
16 20 24 
28 32 36 
M1+M2:
Error: Incompatible dimensions
